### DISCUSSION TITLE
Help please

### DISCUSSION THREAD

**Discussion by Nojarosann** [TheLastBen/fast-stable-diffusion#1777] [Q&A]
This keeps appearing and i have no idea why
![Screenshot 2023-03-15 at 11-23-44 Google Colaboratory](https://user-images.githubusercontent.com/100730490/225338141-989a7db4-404f-449b-83b4-d47ac5f96927.png)

**Comment by TheLastBen** [comment]
a google drive issue, but I don't think it affects the webui

**Reply by Nojarosann** [reply]
then why is it not working? sorry for the bother i'm not a programmer or anything

**Reply by TheLastBen** [reply]
try running the colab with temporary storage, that's by skipping the first cell

**Reply by Nojarosann** [reply]
Can i use finetuned models like that?

**Reply by TheLastBen** [reply]
yes

**Reply by Nojarosann** [reply]
btw i know this is very very late but is there a fix to this or will it fix itself with time?

Edit. i tried giving it (gdrive)link to teh model and checking the "Use temp storage" box but it seems to have given out the same error

**Reply by TheLastBen** [reply]
I'm still investigating
